src/Pages/Profile/admin/SendUnsubscribeEmail.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import './AdminDashboard.css';
+import Header from '../../../Components/Header/Header';
+import { Button, Container } from 'reactstrap';
+import { getAllUserSubscribedAndVerified } from '../../../APIFunctions/User';
+
+function AdminDashboard(props) {
+  const [buttonText, setButtonText] = useState('Send Unsubscribe Email to All');
+  const [buttonColor, setButtonColor] = useState('');
+
+  const handleButtonClick = async () => {
```

Comment:
similar here, we could name this something that's indicative of the function being performed by the button click